.travis.yml

```diff
@@ -26,7 +24,7 @@ install:
   - "make requirements"
   - "pip install coveralls"
 script:
-  - make test
+  - tox -e $TOXENV
```

Comment:
I think that this can be changed with just `tox`

Reply:
done/